src/Symfony/Component/VarDumper/Tests/Caster/ReflectionCasterTest.php

```diff
@@ -227,6 +228,64 @@ public function testReflectionParameterNullableUnion()
         );
     }
 
+    public function testReflectionPropertyScalar()
```

Comment:
This test will fail on PHP 7 because your new fixture class cannot be parsed by a PHP 7 interpreter. Either flag this test as `@requires PHP 8` or create a separate fixture for PHP 7.

Reply:
Yup, I noticed that in CI, I will make this change.